apache/superset
# fix: Add database search in available charts on dashboard.

### SUMMARY

On dashboard, when we try to add a new chart, we can see only 200 charts in chart list section. Because we are fetching charts by limitation of amount of 200. 

And `filtering chart`, `sort by` option only works in frontend, not against database. So we can't see old charts if we have more than 200 charts in database. 

#### Solution.
Need to make `filter/sort by` option work against database.



### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS


1. Go to one dashboard page.
2. Click edit
3. Go go `charts` tab in crud pane.
4. Try to filter chart or sort chart.

Result: You should be able to see that it hits the endpoint, reloads charts data again.

### ADDITIONAL INFORMATION
